.githooks/pre-commit

```diff
@@ -1,4 +1,11 @@
 $!/usr/bin/env bash
 
-echo "==> Running pre-commit git hook..."
-make fmt check
+STAGED_GO_FILES=$(git diff --cached --name-only | grep ".go$")
+
+for FILE in $STAGED_GO_FILES
```

Comment:
[qa] Does this piece of code format your files before the commit?

Reply:
yes, if you install it, it was taking a lot of time in my machine so I changed to only do it for staged files, should improve speed a little